model/converter/json/from_domain.go

```diff
@@ -22,18 +22,29 @@ package json
 
 import (
 	"github.com/uber/jaeger/model"
-
 	"github.com/uber/jaeger/model/json"
 )
 
 // FromDomain converts model.Trace into json.Trace format.
 // It assumes that the domain model is valid, namely that all enums
 // have valid values, so that it does not need to check for errors.
 func FromDomain(trace *model.Trace) *json.Trace {
-	return fromDomain{}.fromDomain(trace)
+	fd := fromDomain{}
+	fd.convertKeyValuesFunc = fd.convertKeyValues
+	return fd.fromDomain(trace)
+}
+
+// FromDomainEmbedProcess converts model.Span into json.Span format.
+// This format includes a ParentSpanID and an embedded Process.
+func FromDomainEmbedProcess(span *model.Span) *json.Span {
```

Comment:
s//SpanFromDomainEmbedProcess/

I would rather pass "embed process" as an option, but we can do it separately.